### ISSUE TITLE
Lemon reset classes not being rendered

### ISSUE THREAD

**Issue by am80l** [Yelp/lemon-reset#44]
Lemon reset classes are not being rendered on any components in my project. I was very confused by this because I was certain I had already verified v1.1.1 as working, but something has gone sideways since then, which has me horribly confused.

To reduce noise, I've set up a basic create-react-app that reproduces the issue. 

https://github.com/am80l/create-react-app-example.git

After setting up create-react-app I then modify the generic app.js to use lemon-reset. 

```js
import React, { Component } from 'react';
import logo from './logo.svg';
import './App.css';
import { Div, Header, Img, H1, P } from 'lemon-reset';

class App extends Component {
  render() {
    return (
      <Div className="App">
        <Header className="App-header">
          <Img src={logo} className="App-logo" alt="logo" />
          <H1 className="App-title">Welcome to Hello world!</H1>
        </Header>
        <P className="App-intro">
          To get started, edit <code>src/App.js</code> and save to reload.
        </P>
      </Div>
    );
  }
}

export default App;
```

As you can see in the screenshot, the classes (and therefore resets) have not been applied.

![screen shot 2018-06-15 at 4 40 27 pm](https://user-images.githubusercontent.com/4461281/41493322-df083cee-70ba-11e8-9744-484fbe9ef5b7.png)

Digging into the React dev tools, we can see that Lemon components are rendering, the stylesheet is also present in the head with the reset class definitions. Every aspect seems to be working, save for the piece that actually puts the lemon class on the html element. 

![screen shot 2018-06-15 at 4 41 33 pm](https://user-images.githubusercontent.com/4461281/41493391-a45db654-70bb-11e8-8bfe-818db8544e3f.png)

![screen shot 2018-06-15 at 4 46 53 pm](https://user-images.githubusercontent.com/4461281/41493418-f2dafa80-70bb-11e8-8175-1888ce967f61.png)

Any help on this would be greatly appreciated, thank you!

Node Version: v8.11.3
NPM Version: v5.6.0

React Version: 16.0.0
React-Dom Version: 16.0.0

(FYI I also updated to React/React-Dom v16 latest which had no effect on the issue)

**Comment by am80l** [comment]
Okay, I was able to get to the bottom of this issue. I also realized create-react-app does not have OOTB support for css modules. Once I went through the steps of ejecting and editing the config to allow css-modules, lemon-reset went back to working as expected. 

My best guess of what happened is when lemon-reset removed the scss loader dependency I removed it from my branch also, and thought I had verified it as still working. I left that code in branch for awhile while focusing on other things, its also completely possible I just missed the issue. But it came down to how I was using the css loader without options:

```{ test: /\.css$/, use: [{ loader: 'style-loader' }, { loader: 'css-loader' }] }```

We rarely dust off this loader since our own project uses a .pcss extension to distinguish postcss files from normal css. I combined this rule with with my existing postcss rules and everything went back to working. 

My guess is lemon-reset requires the options with localIdentName to be configured or it just poops out silently.

```js      { test: /\.css$/, use: [{ loader: 'style-loader' }, { loader: 'css-loader' }] },
      { test: /\.pcss$/, use: [
        'style-loader', {
          loader: 'css-loader',
          options: {
            modules: true,
            localIdentName: 'sh-[local]-[hash:base64:5]',
            importLoaders: 1,
          },
        },
        'postcss-loader',
      ] },
```

If it turns our it to be the case that localIdentName is required, I think it would be nice to see some example loader configuration to avoid confusion. Once I've verified this is the case, I will open up a PR.

**Comment by am80l** [comment]
Okay, I was able to get to the bottom of this issue. I also realized create-react-app does not have OOTB support for css modules. Once I went through the steps of ejecting and editing the config to allow css-modules, lemon-reset went back to working as expected. 

My best guess of what happened is when lemon-reset removed the scss loader dependency I removed it from my branch also, and thought I had verified it as still working. I left that code in branch for awhile while focusing on other things, its also completely possible I just missed the issue. But it came down to how I was using the css loader without options:

```{ test: /\.css$/, use: [{ loader: 'style-loader' }, { loader: 'css-loader' }] }```

We rarely dust off this loader since our own project uses a .pcss extension to distinguish postcss files from normal css. I combined this rule with with my existing postcss rules and everything went back to working. 

```js      { test: /\.css$/, use: [{ loader: 'style-loader' }, { loader: 'css-loader' }] },
      { test: /\.pcss$/, use: [
        'style-loader', {
          loader: 'css-loader',
          options: {
            modules: true,
            localIdentName: 'sh-[local]-[hash:base64:5]',
            importLoaders: 1,
          },
        },
        'postcss-loader',
      ] },
```

My guess is lemon-reset requires the options with modules:true, which arguably should be inferred by the lemon-reset's projects description, though I still think this would be something worth documenting to avoid confusion.

**Comment by am80l** [comment]
Okay, I was able to get to the bottom of this issue. I also realized create-react-app does not have OOTB support for css modules. Once I went through the steps of ejecting and editing the config to allow css-modules, lemon-reset went back to working as expected. 

My best guess of what happened is when lemon-reset removed the scss loader dependency I removed it from my branch also, and thought I had verified it as still working. I left that code in branch for awhile while focusing on other things, its also completely possible I just missed the issue. But it came down to how I was using the css loader without options:

```{ test: /\.css$/, use: [{ loader: 'style-loader' }, { loader: 'css-loader' }] }```

We rarely dust off this loader since our own project uses a .pcss extension to distinguish postcss files from normal css. I combined this rule with with my existing postcss rules and everything went back to working. 

```js      { test: /\.css$/, use: [{ loader: 'style-loader' }, { loader: 'css-loader' }] },
      { test: /\.pcss$/, use: [
        'style-loader', {
          loader: 'css-loader',
          options: {
            modules: true,
            localIdentName: 'sh-[local]-[hash:base64:5]',
            importLoaders: 1,
          },
        },
        'postcss-loader',
      ] },
```

My guess is lemon-reset requires the options with modules:true, which arguably should be inferred by the lemon-reset's projects description, though I still think this would be something worth documenting to avoid confusion.
